sktime/transformations/series/compose.py

```diff
@@ -96,3 +97,204 @@ def inverse_transform(self, Z, X=None):
         if not self.passthrough:
             z = self.transformer_.inverse_transform(z, X=None)
         return z
+
+
+class ColumnComposition(_SeriesToSeriesTransformer):
```

Comment:
We need to think about the name, `ColumnComposition` seems very general. We have a `RowwiseTransformer`, so perhaps `ColumnwiseTransformer`? Or `ColumnApplier` or `ColumnApplyTransformer`?